src/pmpo_fortran.f90

```diff
@@ -209,29 +258,91 @@ subroutine polympo_setMeshNumEdgesPerElm(mpMesh, nCells, nEdgesOnCell) &
     type(c_ptr), intent(in), value :: nEdgesOnCell
   end subroutine
   !---------------------------------------------------------------------------
-  !> @brief set the velocity mesh array from a host array
+  !> @brief set the polympo mesh vertices coordinates
```

Comment:
We could note in the docstrings for the topology modifying functions something like: 
`modifies mesh topology, see polympo_startMeshFill(...)`

Reply:
https://github.com/SCOREC/polyMPO/commit/27059d373dd03d77f998f7224de6795c48afc8de#